ona_migration_script/migrate_submissions.py

```diff
@@ -0,0 +1,44 @@
+from ona import OnaApiClient
+import settings
+
+FROM_FORM = '15756'  # imaliyethu
+TO_FORM = '23666'  # per report
+
+DEFAULTS = {'msisdn': '', 'fault_status': '', 'logged_date': ''}
+
+client = OnaApiClient(settings.USERNAME, settings.PASSWORD)
```

Comment:
I've added username and pass to be taken in the command line (so that the pass doesn't show when typing). Do you think it should go in the args instead?